src/test/java/org/folio/rest/api/ItemDamagedStatusAPITest.java

```diff
@@ -0,0 +1,247 @@
+package org.folio.rest.api;
+
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.folio.rest.support.http.InterfaceUrls.itemDamagedStatusesUrl;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.UUID;
+
+import org.folio.HttpStatus;
+import org.folio.rest.impl.ItemDamagedStatusAPI;
+import org.folio.rest.jaxrs.model.ItemDamageStatus;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.vertx.core.json.JsonObject;
+
+public class ItemDamagedStatusAPITest extends TestBase {
+
+  public static final String TEST_TENANT = "test_tenant";
+
+  @Before
+  public void beforeEach() {
+    StorageTestSuite.deleteAll(TEST_TENANT, ItemDamagedStatusAPI.REFERENCE_TABLE);
+  }
+
+  @Test
+  public void canCreateItemDamagedStatus() {
+
+    ItemDamageStatus status = new ItemDamageStatus()
+      .withName("test item damaged status name")
+      .withSource("local");
+
+    client.post(itemDamagedStatusesUrl(EMPTY), status, TEST_TENANT)
+      .whenComplete((response, throwable) -> {
+        assertThat(response, notNullValue());
+        assertThat(response.getStatusCode(), is(HttpStatus.HTTP_CREATED.toInt()));
+        JsonObject statusJson = response.getJson();
+        assertThat(statusJson.getString("id"), notNullValue());
+        assertThat(statusJson.getString("name"), is(status.getName()));
+      });
+  }
+
+  @Test
+  public void shouldNotCreateItemDamagedStatusWhenUnprocessableEntity() {
```

Comment:
What kind of unprocessable entity?

The body of the test appears to describe when unexpected properties are included in the request, maybe the test could be renamed?
```suggestion
  public void shouldNotCreateItemDamagedStatusWhenUnexpectedPropertiesAreProvided() {
```

Reply:
Done